sotawhat.py

```diff
@@ -5,6 +5,7 @@
 import enchant
 from nltk.tokenize import word_tokenize
 from six.moves.html_parser import HTMLParser
+from termcolor import colored
```

Comment:
```python
$ python2
Python 2.7.15rc1 (default, Apr 15 2018, 21:51:34) 
[GCC 7.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import termcolor
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named termcolor
>>> quit()
$ python
Python 3.7.0 (default, Jun 28 2018, 10:04:41) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import termcolor
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'termcolor'

```

`termcolor` is a third-party package that would have to be added to `requirements.txt`
* https://pypi.org/project/termcolor/